mods/ra2/rules/soviet-infantry.yaml

```diff
@@ -92,12 +92,12 @@ shk:
 	Inherits: ^Infantry
 	Buildable:
 		Queue: Infantry
-		BuildAtProductionType: Infantry
```

Comment:
That sounds like issues with those actors. I can fix them or you can do it in another PR if you'd like.